data/members/accutivesecurity.yaml

```
@@ -16,9 +16,9 @@ sponsor:
   level: 
 
 # Blog posts
-blog: 
-  url: 
-  feed: https://accutivesecurity.com/feed/
+blog:
+  url: https://accutivesecurity.com/insights/
+  feed: https://accutivesecurity.com/category/articles/
```

This does not seem to be a machine readable feed, you want
```suggestion
  feed: https://accutivesecurity.com/category/articles/feed/
```